app.js

```diff
@@ -0,0 +1,61 @@
+/* jshint esversion:6*/
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const app = express();
+const json = bodyParser.json();
+const urlParser = bodyParser.urlencoded({ extended: false });
+const fail = {success: false};
+const success = {success:true};
+const heard = {heard: false};
+const bigBuzz = [];
+let userScore = {score : 0};
+
+app.use(bodyParser.urlencoded());
+
+app.use(express.static('public'));
+
+app.get('/', (request, response)=>{
+  response.render('index');
+
+});
+
+
+let buzzwordObject = {
+  buzzwords: []
+};
+
+
+app.get('/buzzwords', (request, response)=>{
+  response.json(buzzwordObject);
+});
+
+app.post('/buzzwords', urlParser, (request, response)=>{
```

Comment:
urlParser is not needed here since app.use is  already applying bodyPArser across all incoming requests. This is the benefit of using app.use.